src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -4376,11 +4361,8 @@ public Long xlen(final String key) {
     }
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override
-  public long xack(final String key, final String group, final StreamEntryID... ids) {
+  public Long xack(final String key, final String group, final StreamEntryID... ids) {
```

Comment:
So my view in Java is to always prefer primitives unless you need the `null` part, can this one return null? (same comment applies to all the other ones below)

Reply:
@mina-asham I was discussing this with @gkorland just today :)
Please review this PR ignoring this issue.